MonoGame.Framework.Content.Pipeline/Serialization/Compiler/TextureCubeWriter.cs

```diff
@@ -20,6 +20,8 @@ protected internal override void Write(ContentWriter output, TextureCubeContent
             if (!level0.TryGetFormat(out format))
                 throw new Exception("Couldn't get format for TextureCubeContent.");
 
+            format = value.GetActualFormat(format);
```

Comment:
> This is why I originally was looking for a solution that didn't involve adding this to the pipeline.

Yep, and having implemented [a version of] the content pipeline changes, I'm more inclined to that way of thinking too. But I keep coming back to: for any given texture, it either is or it isn't sRGB-encoded, so it makes sense to specify it once at content build time, and let the runtime figure out whether it's supported by the hardware.